README.adoc

```diff
@@ -99,21 +146,44 @@ Click the following button to view the Event Manager application:
 endif::[]
 The event application does not display any events because no events are stored in the database. Go ahead and click `Create Event`, located in the left navigation bar. After entering an event name, location and time, click `Submit` to persist your event entity to the database. The event is now stored in the database and is visible in the list of current events.
```

Comment:
I think we should mention somewhere that the event date should be in the future. The error message is just "Choose a valid time" which doesn't indicate how to fix it.